main/EI 78/CAEM2.sql

```diff
@@ -0,0 +1,32 @@
+SELECT     R.CUIIO,
+           R.CUIIO_VERS,
+           R.CAEM2
+          
+      FROM (
+      
+      
+      SELECT FC.CUIIO,
+                   FC.CUIIO_VERS,
+                   FC.FORM,
+                   FC.FORM_VERS,
```

Comment:
SELECT 
    R.CUIIO,
    R.CUIIO_VERS,
    R.CAEM2
FROM 
    (SELECT 
        FC.CUIIO,
        FC.CUIIO_VERS,
        FC.FORM,
        FC.FORM_VERS,
        FC.STATUT
     FROM 
        CIS2.FORM_CUIIO FC
     INNER JOIN 
        (SELECT 
            CUIIO, 
            MAX(CUIIO_VERS) AS CUIIO_VERS
         FROM 
            CIS2.FORM_CUIIO
         WHERE 
            FORM IN (:pFORM) 
            AND CUIIO_VERS <= :pPERIOADA
         GROUP BY 
            CUIIO) BB
     ON 
        BB.CUIIO = FC.CUIIO
        AND BB.CUIIO_VERS = FC.CUIIO_VERS
     WHERE 
        FC.FORM IN (:pFORM) 
        AND FC.STATUT <> '3'
    ) FC
INNER JOIN 
    CIS2.RENIM R
ON 
    R.CUIIO = FC.CUIIO 
    AND R.CUIIO_VERS = FC.CUIIO_VERS
WHERE 
    R.CAEM2 IS NULL;